drivers/pinctrl/pinctrl_lpc_iocon.c

```diff
@@ -11,23 +11,40 @@
 #include <fsl_clock.h>
 #endif
 
+/* RT7xx requests to clear the reset bit in RESET module before a IOPCTL can be used. */
+#if defined(CONFIG_SOC_SERIES_IMXRT7XX)
```

Comment:
So right now we are essentially using a different mask definition for the RT700? I think we should probably use some DT property on the iocon node to control if we use this mask definition or not, or we should transition *all* SOCs using this pin controller to use the new macro definition.

@decsny any thoughts here on the usage of DT to control this behavior?